app/build.gradle.kts

```diff
@@ -38,7 +38,7 @@ android {
     namespace = "io.element.android.x"
 
     defaultConfig {
-        applicationId = "io.element.android.x"
+        applicationId = "ashlee.animent.android.x"
```

Comment:
It looks like you are creating an alternative application. That's fine, but the PR does not need to exist.